doc/netplan.md

```diff
@@ -79,6 +79,10 @@ Virtual devices
 
 ## Common properties for physical device types
 
+     **Note:** These options will not work reliably for devices matched by name
```

Comment:
@rlaager and @slyon :

I moved this note a little up so that it generally refers to options rendered in the `.link` file.

Reply:
I made this comment a bit more specific, as it does not apply to all of the "Common properties for physical device types", but only the udev/.link changes. And actually I added some `udevadm test /sys/class/net/IFACE` command to `netplan apply`, which might resolve this issue in some cases. At least `netplan apply` is now able to apply the `*-offload` options at runtime (without reboot) during the integration tests. cc @rlaager